example/src/main/webapp/geodesk/index.html

```diff
@@ -1,19 +1,29 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+<!DOCTYPE HTML>
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="nl" lang="nl">
 <head>
+	<script>isc_css3Mode = "on";</script>
     <meta http-equiv="content-type" content="text/html;charset=utf-8" />
-    <!-- Mimic Internet Explorer 7 -->
-    <meta http-equiv="X-UA-Compatible" content="IE=7" >
+    <meta name="gwt:property" content="locale=nl">
+    <meta http-equiv="X-UA-Compatible" content="IE=10" >
```

Comment:
possibly, even <meta http-equiv="X-UA-Compatible" content="IE=10" > can be removed